core/assets/images.py

```diff
@@ -0,0 +1,41 @@
+from enum import Enum
+import pygame
+from core.assets.assets import Assets, CustomAssets
+from core.assets.image import Image
+
+
+class Images(Enum):
```

Comment:
Remove deze class en stop het in assets.py, al doe je het in een aparte section

Reply:
al gedaan in 'beter gemaakt naar Melles wil'